vagrant-vbguest with Oracle Linux 8

Box:  https://yum.oracle.com/boxes/oraclelinux/latest/ol8-latest.box
VirtualBox: 6.1
Vagrant: 2.2.6
Plugin: vagrant-vbguest (0.22.1, global)


While creating a new deployment using Oracle Linux 8, I noticed the guest additions build failed with the following error:

$ tail -n 20 /var/log/vboxadd-setup.log
Building the main Guest Additions 6.1.0 module for kernel 4.18.0-147.3.1.el8_1.x86_64.
Error building the module.  Build output follows.
make V=1 CONFIG_MODULE_SIG= CONFIG_MODULE_SIG_ALL= -C /lib/modules/4.18.0-147.3.1.el8_1.x86_64/build M=/tmp/vbox.0 SRCROOT=/tmp/vbox.0 -j2 modules
make[1]: warning: -jN forced in submake: disabling jobserver mode.
Makefile:977: *** "Cannot generate ORC metadata for CONFIG_UNWINDER_ORC=y, please install libelf-dev, libelf-devel or elfutils-libelf-devel".  Stop.
make: *** [/tmp/vbox.0/Makefile-footer.gmk:114: vboxguest] Error 2
Could not find the X.Org or XFree86 Window System, skipping.
modprobe vboxguest failed

Installed the following package:  $ sudo dnf isntall elfutils-libelf-devel
Rebuilt additions using: $ sudo /usr/sbin/rcvboxadd setup

Success!!!

After this change, the guest additions installed on Oracle Linux 8 successfully.

--- a/lib/vagrant-vbguest/installers/oracle.rb
+++ b/lib/vagrant-vbguest/installers/oracle.rb
@@ -15,7 +15,8 @@ module VagrantVbguest
           'gcc',
           'make',
           'perl',
-          'bzip2'
+          'bzip2',
+          'elfutils-libelf-devel'
         ].join(' ')
       end
     end

